### PR TITLE
Login method updated with dataInterface

### PR DESCRIPTION
**Pull request by KasperSch**
Login method updated to work with new DataInterface

### CODE REVIEW

File: src/domain/System.java [hunk 1]
```diff
@@ -51,8 +51,8 @@ protected static SuperUser getSuperUser() {
 
 
     //missing failure message when login is incorrect
-    public Boolean logIn(String inputUsername, String inputPassword) {
```
Comment (fusuf) [inline src/domain/System.java:54]: its boolean not Boolean

**Review by fusuf — CHANGES_REQUESTED** [review]

**Comment by KasperSch** [comment]
nej

File: src/domain/System.java [hunk 2]
```diff
@@ -51,8 +51,8 @@ protected static SuperUser getSuperUser() {
...
-        this.superUser = dataManager.checkIfUserExists(inputUsername, inputPassword);
+    public static boolean logIn(String inputUsername, String inputPassword) {
```
Comment (fusuf) [inline src/domain/System.java:54]: static no

**Review by fusuf — CHANGES_REQUESTED** [review]
GG

**Review by fusuf — APPROVED** [review]
lol

**Pull request merged**